pkg/cmd/integration.go

```diff
@@ -264,22 +268,65 @@ oc plugin mobile delete integration <consuming_service_instance_id> <providing_s
 			if err != nil {
 				return errors.WithStack(err)
 			}
-			if !redeploy {
-				fmt.Println("you will need to redeploy your service to pick up the changes")
+			noWait, err := cmd.PersistentFlags().GetBool("no-wait")
+			if err != nil {
+				return errors.WithStack(err)
+			}
+			if noWait && !redeploy && !quiet {
```

Comment:
I think quiet should hide non essential information, I feel like this is too important to not show